lib/mpl_toolkits/mplot3d/spherical.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+
+# This module adds functionality to plot spherical polygons and is a result of
+# the discussion of issue #5294 (Handling Spherical Polygons) on github. The
+# following people contributed to this discussion:
+# Nikolai Nowaczyk, Tyler Reddy, Nicolas P. Rougier, Phillip Wolfram,
+# Thomas A Caswell
+
+import numpy as np
+from mpl_toolkits.mplot3d.art3d import Poly3DCollection
+
+
+class SphericalPolygon(object):
+    """
+     Plots a spherical polygon onto a sphere.
+    """
+
+    def __init__(self,
+                 vertices,
+                 tri=[[0, 1, 2]],
+                 center=np.array([0, 0, 0]),
+                 radius=1.,
+                 subdivisions=None):
+        """
+        Parameters
+        ----------
+        vertices: an array of points in 3D assumed to be on a sphere
+        tri: a list [[i,j,k], ...] of lists [i,j,k] of integers i,j,k, where
```

Comment:
No particular reason - probably equivalent in most situations. Thinking about it in more detail I agree that one should use the NumPy array here, because it has better performance on large instances and because the triangulation returned by `scipy.spatial.ConvexHull` is also a NumPy array of integers.